packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -277,16 +272,16 @@ export class UIFormComponent extends React.Component {
 
 
 	render() {
-		const { onEnterSubmit, onLeaveSubmit } = this.props;
+		const { onSubmitEnter, onSubmitLeave } = this.props;
 		const actions = this.props.actions || [
 			{
 				bsStyle: 'primary',
 				label: 'Submit',
 				type: 'submit',
 				widget: 'button',
 				position: 'right',
-				onMouseEnter: onEnterSubmit && this.onSubmitHover,
-				onMouseLeave: onLeaveSubmit,
+				onMouseEnter: onSubmitEnter && this.onSubmitHover,
```

Comment:
`const { properties, onSubmitEnter, onSubmitLeave } = this.props;`

`onSubmitEnter && event => onSubmitHover(event, properties)`